examples/test-no-results.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="utf-8">
+
+		<!--
+		Always force latest IE rendering engine (even in intranet) & Chrome Frame
+		Remove this if you use the .htaccess
+		-->
+		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+		<title>Universal Language Selector</title>
+		<!-- <link rel="shortcut icon" href="/favicon.ico"> -->
+		<!-- <link rel="apple-touch-icon" href="/apple-touch-icon.png"> -->
+		<!-- <meta name="description" content=""> -->
+		<meta name="author" content="Santhosh Thottingal">
+		<link href="../css/jquery.uls.css" rel="stylesheet">
+		<link href="../css/jquery.uls.grid.css" rel="stylesheet">
+		<link href="../css/jquery.uls.lcd.css" rel="stylesheet">
+		<!-- demo -->
+		<link href="resources/demo.css" rel="stylesheet">
+		<!-- Libs -->
+		<script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+		<!-- Source -->
+		<script src="../src/jquery.uls.data.js"></script>
+		<script src="../src/jquery.uls.data.utils.js"></script>
+		<script src="../src/jquery.uls.lcd.js"></script>
+		<script src="../src/jquery.uls.languagefilter.js"></script>
+		<script src="../src/jquery.uls.core.js"></script>
+		<script>
+			$( document ).ready( function() {
+				$( '.uls-trigger' ).uls( {
+					onSelect : function( language ) {
+						var languageName = $.uls.data.getAutonym( language );
+						$( '.uls-trigger' ).text( languageName );
+					},
+					noResultsTemplate: '<div>No article exist in the language you searched</div>'
+				} );
+			} );
+		</script>
```

Comment:
This demo doesn't really illustrate the difference between "unknown language" and "no search results".

Reply:
Updated the demo to use the current search query